wannierberri/system/sym_wann.py

```diff
@@ -100,16 +107,20 @@ def __init__(
             '_WCC': 1,
             'BB': 1,
             'CC': -1,
-            'SS': -1
-        }  # {'AA':1,'BB':1,'CC':1,'SS':-1,'SA':1,'SHA':1,'SR':1,'SH':1,'SHR':1}
+            'SS': -1,
+            'SA': 1, 'SHA': 1, 'SR': 1, 'SH': 1, 'SHR': 1,
```

Comment:
If I change any of the signs in this line, the test fails. Which suggests that the signs are correct, but ...